### PR TITLE
Update the project file and NuGet packages.

### PR DESCRIPTION
**Pull request by teo-tsirpanis**
This PR:

* Adds a .NET Standard 2.0 target for DiffPlex, as [it is recommended](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting).
* Enables SourceLink (you have to publish the `snupkg` files as well).
* Use SPDX license expressions instead of including the license file to the NuGet packages.
* Updates the test framework to .NET Core 3.1.
* Updates the WebDiffer sample site to ASP.NET Core 3.1.

### CODE REVIEW

File: DiffPlex/DiffPlex.csproj [hunk 1]
```diff
@@ -1,32 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\NuGet.props" />
   <PropertyGroup>
-    <TargetFrameworks>net35;net40;netstandard1.0</TargetFrameworks>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <TargetFrameworks>net35;net40;netstandard1.0;netstandard2.0</TargetFrameworks>
```
Comment (mmanela) [inline DiffPlex/DiffPlex.csproj:4]: Should we drop the netstandard1. 0 target based on the link you provided?

File: DiffPlex/DiffPlex.csproj [hunk 2]
```diff
@@ -1,32 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\NuGet.props" />
```
Comment (mmanela) [inline DiffPlex/DiffPlex.csproj:2]: Awesome thanks for making the props file

**Comment by mmanela** [comment]
How does publishing the snupkg work? Is it a separate nuget package?

File: DiffPlex/DiffPlex.csproj (hunk 1 above)
Comment (teo-tsirpanis) [reply DiffPlex/DiffPlex.csproj:4]: It says _avoid_, not don't. Targeting .NET Standard 1.0 makes DiffPlex compatible with older platforms like .NET Core 1.x.

DiffPlex doesn't use a class that doesn't support .NET Standard 1.0 at the moment, so keeping it doesn't hurt anyone.

**Comment by teo-tsirpanis** [comment]
> Is it a separate nuget package?

Essentially yes. [You have to publish it separately to NuGet](https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg).

**Comment by mmanela** [comment]
Looks good. Thanks!

**Pull request merged**